node/process.ts

```diff
@@ -345,6 +345,9 @@ class Process extends EventEmitter {
    */
   argv = argv;
 
+  /** https://nodejs.org/api/process.html#process_process_execargv */
+  execArgv = [];
```

Comment:
We don't expose these arguments in Deno, so effectively this array will always be empty. I noticed some projects rely on it to sniff out if Node was run with `--inspect` flag. I guess we just have to roll with this solution for now

Reply:
I agree this is enough for now